src/table/TableBodyWrapper.jsx

```diff
@@ -65,10 +108,21 @@ const TableBodyWrapper = ({
 
   useEffect(() => {
     addSelectionListeners(selectionsAPI, selDispatch, setShouldRefocus);
+    const bodyElement = document.getElementsByTagName('tbody')[0];
+    const showScroll = () => setIsScrollbar(true);
+    const hideScroll = () => setIsScrollbar(false);
+    bodyElement.addEventListener('mouseover', showScroll, true);
+    bodyElement.addEventListener('mouseleave', hideScroll, true);
+    return () => {
+      window.removeEventListener('mouseover', showScroll);
+      window.removeEventListener('mouseleave', hideScroll);
+    };
```

Comment:
Isn't this the same thing as just using the hover pseudo-class?

Reply:
when using the hover pseudo-class, can we change the style of the scrollbar? Look into that.

Reply:
yeah maybe hard to set `&&::-webkit-scrollbar-track` etc